raiden/tests/unit/test_utils.py

```diff
@@ -92,3 +93,8 @@ def request_mock(method, url, **_):
 
     with patch.object(requests, "request", side_effect=request_mock):
         assert get_http_rtt(url="url", method="get") == 0.3
+
+
+def test_pack_data():
```

Comment:
Could you add some happy path tests as well?

Reply:
Added a happy path and another failing path.